Editors/README.md

```diff
@@ -29,6 +29,10 @@ After installing the extension, settings for SourceKit-LSP can be found in `Pref
 * `sourcekit-lsp.toolchainPath`: (optional) The path of the swift toolchain (sets `SOURCEKIT_TOOLCHAIN_PATH`). By default, sourcekit-lsp uses the toolchain it is installed in.
 * `sourcekit-lsp.tracing.server`: Traces the communication between VS Code and the SourceKit-LSP language server
 
+## Atom
+
+Download the `ide-sourcekit` package for Atom from [a corresponding package page](https://atom.io/packages/ide-sourcekit). It also contains installation instructions to get you started.
```

Comment:
Gramatically it should be "**the** corresponding package page". Otherwise LGTM

Reply:
I wanted to change it. But Github's suggestion mechanism did not allow me.

Reply:
My bad, thanks for the correction 🙏